naughty/arch/5881-curl-ssl-errno-0

```diff
@@ -0,0 +1,3 @@
+Traceback (most recent call last):
+  File "*/check-machines-create", line *, in testCreateUrlSource
+    runner.createTest(TestMachinesCreate.VmDialog(self, sourceType='url',
```

Comment:
Can we make that more specific somehow? Otherwise this would match whenever anything is broken with `createTest`, not just when it is curls fault.

Reply:
E.g., there is also "CURL: Error opening file: OpenSSL SSL_read: SSL_ERROR_SYSCALL, errno 0" in the browser log, a few lines above.

Reply:
OK.